src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/InputMethodParamPad.java

```diff
@@ -0,0 +1,83 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.methodparampad;
```

Comment:
@GitToasterhub Yes.
If you look at https://github.com/checkstyle/checkstyle/pull/3983/files , you ca see the file head has the `->` arrow. This signifies the file moved to a new location.